### ISSUE TITLE
Tom Clancy's Splinter Cell (13560)

### ISSUE THREAD

**Issue by ffrasisti** [ValveSoftware/Proton#6435]
# Compatibility Report
- Name of the game with compatibility issues: Tom Clancy's Splinter Cell
- Steam AppID of the game: 13560

## System Information
Steam Deck. 
Latest stable SteamOS
- Proton version: 7.0.5

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
Black textures on characters and enviroment in certain cutscenes. 
And ocasionally when first loading a save after starting a session the enviroment will have no textures
The cutscene problem did not exist in Proton 6.3-8, making this a regression
It has a workaround , using "-shadowmode=projector"  in launch parameters, but this introduces a few shadow problems of its own. 

The no enviroment textures after loading,  if the level is loaded from a save after a cold start of the game, the textures of the level may be black - this is fixed by loading the level "Police Station" and then loading your save. I got this info from protondb as otherwise I could not play. Aside from this annoyances the games works wonderfully with the pc Widescreefix so I hope this can be sorted out

## Reproduction
Try any current version of proton, launch the game and watch the first real time cutscene the plays after loading the Training Level. 

Save on a different level. Restar the deck and the game, load the save, chance of no enviroment textures.

**Comment by kisak-valve** [comment]
Hello @ffrasisti, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the regression, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

You mentioned a third party modification. It would be nice if it was made explicitly clear where you tested the unmodified game versus modified.

**Comment by ffrasisti** [comment]
I first tested it without any changes and it is the same. Without the launch command, you can only avoid issues on proton 6. Can i do that logging on steam deck? I dont have a linux desktop and my deck is only 64gb so if the log is big i might not be able to get it. Would screenshots help in anyway?

**Comment by alasky17** [comment]
@ffrasisti 
Yes it is possible to do the logging on the deck.  You would likely need to switch to desktop mode and then login to github in order to upload here.  However, in this case I've already been able to reproduce the regression, so you got lucky this time and I already got a log :) 

I did not play through the first level to get a save game in order to reproduce the second problem you described though.  I am hopeful that they are the same issue, but if you are able to upload a game save file, that would help me out a lot :)

**Comment by ffrasisti** [comment]
I stopped playing quickly after the report because of work, but I know this works as I've used save files from this site and it even lists the proton path for the save https://www.yoursavegames.com/2019/06/splinter-cell-save-game.html After applying it you can choose any level. I sometimes had problems opening any random later game level and had it load without textures. Loading back to Police station level fixed it. Im guessing its the same as the cutscene problem, though.
Also, shouldn't this have the regression tag applied? Thanks

**Comment by ffrasisti** [comment]
> @ffrasisti Yes it is possible to do the logging on the deck. You would likely need to switch to desktop mode and then login to github in order to upload here. However, in this case I've already been able to reproduce the regression, so you got lucky this time and I already got a log :)
> 
> I did not play through the first level to get a save game in order to reproduce the second problem you described though. I am hopeful that they are the same issue, but if you are able to upload a game save file, that would help me out a lot :)

https://chordian.net/files/saves/Save_Games_SplinterCell1.zip here you go, the other link i showed is dead, this is also a 100% save file from https://blog.chordian.net/save-games-for-pc/

**Comment by alasky17** [comment]
@ffrasisti Thank you for the save!  I haven't had time to test it yet, but the regression (at least with the black figures in cutscenes) is fixed in experimental bleeding-edge (can be selected from Experimental Beta drop-down menu).  If you get a chance to give it a try, please let me know if it also fixes the intermittent save launch failure :)

**Comment by alasky17** [comment]
@ffrasisti The rendering in cutscenes is now fixed in the main experimental branch -- please let me know if you still encounter the occasional rendering failure when opening a save file - for now I'm hoping the regression is fixed unless I hear otherwise :)

**Comment by ffrasisti** [comment]
> @ffrasisti The rendering in cutscenes is now fixed in the main experimental branch -- please let me know if you still encounter the occasional rendering failure when opening a save file - for now I'm hoping the regression is fixed unless I hear otherwise :)

I played a bit more and the problem seems gone. thanks!
